torchinductor/sizevars.py

```diff
@@ -252,7 +253,33 @@ def __getitem__(self, val: int) -> Expr:
     def size_hint(self, expr: Expr) -> int:
         return int(sympy.expand(expr).subs(self.var_to_val))
 
-    def stride_vars(self, index: Expr, vars: List[sympy.Symbol]) -> List[Expr]:
+    def _lru_cache(self, fn, maxsize=None):
```

Comment:
Seems like we could plausibly abstract this into something that can be applied to other methods as well.